packages/mobile/src/components/details-tile/DetailsTileNoAccess.tsx

```diff
@@ -312,80 +326,87 @@ export const DetailsTileNoAccess = ({
     followee,
     tippedUser,
     handlePressCollection,
+    renderLockedSpecialAccessDescription,
     handleFollowArtist,
     handleSendTip,
     styles
   ])
 
-  const renderUnlockingDescription = useCallback(() => {
-    if (nftCollection) {
+  const renderUnlockingSpecialAccessDescription = useCallback(
+    (args: { entity: User; prefix: string; suffix: string }) => {
+      const { entity, prefix, suffix } = args
       return (
         <View style={styles.descriptionContainer}>
           <Text>
-            <Text style={styles.description}>
-              {messages.unlockingCollectibleGatedPrefix}
-            </Text>
+            <Text style={styles.description}>{prefix}</Text>
             <Text
-              onPress={handlePressCollection}
-              style={[styles.description, styles.collectionName]}
+              style={[styles.description, styles.name]}
+              onPress={() => handlePressArtistName(entity.handle)}
```

Comment:
I'm surprised this doesn't throw a lint warning. Does our style guide say anything about arrow functions defined directly in props? This makes a new function every time it's rendered.

Reply:
oh nice catch. going to update